python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -1107,11 +1112,34 @@ def wipe_assets(self, asset_keys):
 
     # event subscriptions
 
-    def get_logger(self):
-        logger = logging.Logger("__event_listener")
-        logger.addHandler(_EventListenerLogHandler(self))
-        logger.setLevel(10)
-        return logger
+    def get_loggers(self):
+
+        loggers = []
+
+        event_logger = logging.Logger("__event_listener")
+        event_logger.addHandler(_EventListenerLogHandler(self))
+        event_logger.setLevel(10)
+        loggers.append(event_logger)
+
+        if self.log_conf:
+            logging.config.fileConfig(self.log_conf)
+
+            with open(self.log_conf, 'r+') as f:
+                lines = f.readlines()
+
+                logger_names = []
+
+                # Searches for logger names within configuration file
```

Comment:
We need to know the names of the python loggers in order to add them to the dagster instance. Currently just parsing the file to retrieve the names, but not sure if there is a better solution.